config/config.php

```diff
@@ -1075,10 +1075,22 @@
 		],
 	],
 	'daily' => [
-		'23' => [
+		'25' => [
 			'downloadUrl' => 'https://download.nextcloud.com/server/daily/latest-master.zip',
 			'web' => 'https://docs.nextcloud.com/server/latest/admin_manual/maintenance/upgrade.html',
 			'eol' => false,
+			'minPHPVersion' => '8.0',
+		],
```

Comment:
```suggestion
		'26' => [
			'downloadUrl' => 'https://download.nextcloud.com/server/daily/latest-master.zip',
			'web' => 'https://docs.nextcloud.com/server/latest/admin_manual/maintenance/upgrade.html',
			'eol' => false,
			'minPHPVersion' => '8.0',
		],
		'25' => [
			'downloadUrl' => 'https://download.nextcloud.com/server/daily/latest-stable25.zip',
			'web' => 'https://docs.nextcloud.com/server/latest/admin_manual/maintenance/upgrade.html',
			'eol' => false,
			'minPHPVersion' => '7.4',
		],
```

Also while on it, can you fix the eol flags of 23 and lower to true?